### ISSUE TITLE
Potentially change sort order in lists

### ISSUE THREAD

**Issue by corneliusroemer** [GenSpectrum/cov-spectrum-website#193]
All the lists on the website are ordered from left to right, then top to bottom, which is not how my eyes naturally read lists with such big spacing between columns. It's much easier to go down with your eyes along columns than go from left to right along rows, at least for me.

Am I alone or do other people feel the same?

I would find it more natural if the sorting was first top to bottom, then left to right:
![image](https://user-images.githubusercontent.com/25161793/124367234-aa27d480-dc55-11eb-8ff5-dc822eeb45a7.png)
![image](https://user-images.githubusercontent.com/25161793/124367236-aac06b00-dc55-11eb-8884-e0e19fd1b55a.png)

**Comment by chaoran-chen** [comment]
I feel it the same way but I'm not sure whether this can be easily implemented due to the way how the browser renders the content.

@TKGZ Do you know how this could be done?

**Comment by spacephoton** [comment]
We could make the lists more dense (closer packed). I wonder if flex-direction might fix? I'll have a look later.

**Comment by corneliusroemer** [comment]
I can manually get it the way we'd want it by using `flex-direction: column`, BUT you need to set the height dynamically, so that you get more than one column - that's not very nice. I don't know how to fix that.

You'd need to calculate how many columns you can fit in, how many elements you have, and then set the height property as a result of that. Not really how CSS is supposed to work, right?
```
.JiLrb {
    display: flex;
    flex-wrap: wrap;
    margin-top: 10px;
    flex-direction: column;
    height: 500px;
```

![image](https://user-images.githubusercontent.com/25161793/124369326-f7fb0780-dc6a-11eb-9370-960f3ba5981f.png)

Maybe there are some more tricks similar to this article: https://css-tricks.com/arranging-elements-top-bottom-instead-left-right-float/

**Comment by corneliusroemer** [comment]
An alternative is to use:
```
ul { column-count: 3;}
li { display: inline-block; }
```
That way you don't have the height problem:
![image](https://user-images.githubusercontent.com/25161793/124369421-1281b080-dc6c-11eb-8cd3-e0c176199dbd.png)

**Comment by chaoran-chen** [comment]
Having to calculate the number of columns/the height is exactly the problem. We would like the website to work on different screen sizes and when the user resizes the window. We do width detection in our `PackedGrid` component:

https://github.com/cevo-public/cov-spectrum-website/blob/62d4bb42b8e471e4b4690efe2549c2c1dfa37010/src/components/PackedGrid/PackedGrid.tsx#L38

but I'm not sure how much effort it will take to use the same technique for this case.

**Comment by chaoran-chen** [comment]
The mutation lists are now in vertical order (fixed in #457).

Because the pango lineage list does not have much vertical space, I think that it's better to keep it the way it is. @corneliusroemer, or do you have some ideas on how to improve the pango lineage list?

![image](https://user-images.githubusercontent.com/18666552/162992415-efffe36f-8104-4b52-aea4-8978938c91f7.png)

**Comment by corneliusroemer** [comment]
It's ok with me - though column major would probably be more consistent.
